internal/storage/bucket_handle.go

```diff
@@ -64,15 +64,12 @@ func (bh *bucketHandle) NewReader(
 	}
 
 	// Creating a NewRangeReader instance.
-	r, err := obj.NewRangeReader(ctx, start, length)
+	rc, err = obj.NewRangeReader(ctx, start, length)
 	if err != nil {
```

Comment:
We don't need this check now. we can remove it.

Reply:
fixed